spec/decanter/decanter_core_spec.rb

```diff
@@ -546,6 +553,28 @@ def self.name
     end
 
     context 'with args' do
+      context 'when strict mode is set to :ignore' do
+        context 'and params include unhandled keys' do
+          let(:decanter) {
+            Class.new(Decanter::Base) do
+              input :name, :string
+              input :description, :string
+            end
+          }
+
+          let(:args) { { name: 'My Trip', description: 'My Trip Description', foo: 'bar' } }
+
+          it 'returns a hash with the declared key-value pairs, ignores unhandled key-vale pairs' do
+            decanter.strict :ignore
+            decanted_params = decanter.decant(args)
+
+            expect(decanted_params).not_to match(args)
+            expect(decanted_params.keys).not_to include([:foo])
+            expect(decanted_params.keys).to eq([:name, :description])
```

Comment:
Would you consider both of these necessary to communicate the feature?

Reply:
the `not_to` case is probably the main one, i'll update.